webpack.profiles.config.js

```diff
@@ -89,16 +89,18 @@ module.exports = function webpack(env = {}, argv = {}) {
             from: 'node_modules/typeface-sedgwick-ave/files/',
             to: path.join(defaultWebpackConfig.output.path, 'fonts/')
           },
-          argv.mode === 'production'
-            ? null
-            : {
-                from: 'test/integration',
-                to: path.join(
-                  defaultWebpackConfig.output.path,
-                  'test',
-                  'integration'
-                )
-              }
+          {
+            from: 'test/integration',
```

Comment:
Maybe we can remove the `test/integration` entry ? 

When is the last time it has been used ?

Reply:
I still need `test/integration/bulle-extension.js` to deploy a fully functionnal environment when it's not published on the Wordpress.